### DISCUSSION TITLE
the picture of the site acces dialog should be updated for every language

### DISCUSSION THREAD

**Discussion by scratchusernamemrtbts** [ScratchAddons/ScratchAddons#1695] [Translation]
the please enable site acces example picture of the dialog should have a picture of every language
i have a picture for thai
<img width="449" alt="ภาพถ่ายหน้าจอ 2564-03-01 เวลา 19 27 23" src="https://user-images.githubusercontent.com/66320961/109497478-b3950a80-7ac4-11eb-941d-468202cd3ae2.png">

**Comment by retronbv** [comment]
I’m sorry this makes no sense? 
> the please enable site acces example picture of the dialog should have a picture of every language
i have a picture for thai

**Comment by mxmou** [comment]
Wouldn't this be too many images?

**Reply by scratchusernamemrtbts** [reply]
I mean that if the browser language is Thai it will show the Thai picture
but if the browser language is English it  will show the English picture

**Comment by TheColaber** [comment]
They are talking about the images located here: https://github.com/ScratchAddons/ScratchAddons/tree/master/images/screenshots

I have forgotten where these images are displayed. Nevertheless, we can do this somehow.

**Reply by WorldLanguages** [reply]
Not sure if we should, tho
